CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## Unreleased
+
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+## v0.10.0
```

Comment:
nit: How about adding a release date here as an example of "Keep a Changelog" does?

Reply:
https://github.com/cookpad/armg/pull/16/commits/aa2d86d5fd7e4962cd27666b717dcd4fb4e68e12